# Process Analysis

In [1]:
python3 ../../volatility3/vol.py -f ../S039/Memory/ACC-03-dc8c73f9.vmem windows.pslist.PsList

Volatility 3 Framework 2.0.0

PID	PPID	ImageFileName	Offset(V)	Threads	Handles	SessionId	Wow64	CreateTime	ExitTime	File output

4	0	System	0xfa8003c78b30	88	525	N/A	False	2021-06-23 13:06:39.000000 	N/A	Disabled
240	4	smss.exe	0xfa80051a5100	2	30	N/A	False	2021-06-23 13:06:39.000000 	N/A	Disabled
320	312	csrss.exe	0xfa80059c4b30	9	599	0	False	2021-06-23 13:06:40.000000 	N/A	Disabled
372	312	wininit.exe	0xfa8005b14350	3	79	0	False	2021-06-23 13:06:40.000000 	N/A	Disabled
384	364	csrss.exe	0xfa80050f76a0	12	433	1	False	2021-06-23 13:06:40.000000 	N/A	Disabled
436	364	winlogon.exe	0xfa8006080b30	3	123	1	False	2021-06-23 13:06:40.000000 	N/A	Disabled
456	372	services.exe	0xfa80060552f0	10	272	0	False	2021-06-23 13:06:40.000000 	N/A	Disabled
488	372	lsass.exe	0xfa800608c910	8	859	0	False	2021-06-23 13:06:40.000000 	N/A	Disabled
496	372	lsm.exe	0xfa8006098b30	10	201	0	False	2021-06-23 13:06:40.000000 	N/A	Disabled
596	456	svchost.exe	0xfa80061799e0	11	370	0	False	2021-06-23 13:06:40.000000 	


Progress:    0.00		Scanning memory_layer using BytesScanner 

Progress:    0.31		Scanning memory_layer using BytesScanner 

Progress:    0.63		Scanning memory_layer using BytesScanner 

Progress:    0.00		Scanning layer_name using PdbSignatureScanner

Progress:    0.00		Scanning layer_name using PdbSignatureScanner

Progress:  100.00		PDB scanning finished                        


In [2]:
python3 ../../volatility3/vol.py -f ../S039/Memory/ACC-03-dc8c73f9.vmem windows.pstree.PsTree

Volatility 3 Framework 2.0.0

PID	PPID	ImageFileName	Offset(V)	Threads	Handles	SessionId	Wow64	CreateTime	ExitTime

4	0	System	0xfa8003c78b30	88	525	N/A	False	2021-06-23 13:06:39.000000 	N/A
* 240	4	smss.exe	0xfa80051a5100	2	30	N/A	False	2021-06-23 13:06:39.000000 	N/A
320	312	csrss.exe	0xfa80059c4b30	9	599	0	False	2021-06-23 13:06:40.000000 	N/A
* 1500	320	conhost.exe	0xfa80066f07d0	2	34	0	False	2021-06-23 13:06:41.000000 	N/A
* 1388	320	conhost.exe	0xfa80066ed060	2	32	0	False	2021-06-23 13:06:41.000000 	N/A
372	312	wininit.exe	0xfa8005b14350	3	79	0	False	2021-06-23 13:06:40.000000 	N/A
* 456	372	services.exe	0xfa80060552f0	10	272	0	False	2021-06-23 13:06:40.000000 	N/A
** 1416	456	VGAuthService.	0xfa80066f50e0	4	98	0	False	2021-06-23 13:06:41.000000 	N/A
** 3852	456	winlogbeat.exe	0xfa8006073230	17	191	0	False	2021-06-23 13:08:48.000000 	N/A
** 1300	456	ssm.exe	0xfa800616cb30	3	56	0	False	2021-06-23 13:06:41.000000 	N/A
*** 1492	1300	python.exe	0xfa80066df060	6	296	0	False	2021-06-23

# Network Analysis

In [3]:
python3 ../../volatility3/vol.py -f ../S039/Memory/ACC-03-dc8c73f9.vmem windows.netstat.NetStat

Volatility 3 Framework 2.0.0

Offset	Proto	LocalAddr	LocalPort	ForeignAddr	ForeignPort	State	PID	Owner	Created

0xfa8006a69ba0	TCPv4	127.0.0.1	49197	127.0.0.1	49198	ESTABLISHED	1492	python.exe	N/A
0xfa8006a67340	TCPv4	127.0.0.1	49195	127.0.0.1	49196	ESTABLISHED	1492	python.exe	N/A
0xfa8006a0fba0	TCPv4	127.0.0.1	49191	127.0.0.1	49192	ESTABLISHED	1492	python.exe	N/A
0xfa8006a3a5e0	TCPv4	172.17.11.190	49293	52.203.247.180	443	ESTABLISHED	3852	winlogbeat.exe	N/A
0xfa8006aa6440	TCPv4	172.17.11.190	49292	10.20.1.27	9001	ESTABLISHED	3748	winlogbeat.exe	N/A
0xfa8006967cf0	TCPv4	127.0.0.1	49183	127.0.0.1	49184	ESTABLISHED	1492	python.exe	N/A
0xfa800422c900	TCPv4	172.17.11.190	55026	10.20.1.11	8000	ESTABLISHED	3692	Velociraptor.e	N/A
0xfa8006a943e0	TCPv4	127.0.0.1	49198	127.0.0.1	49197	ESTABLISHED	1492	python.exe	N/A
0xfa800545ca00	TCPv4	172.17.11.190	55038	172.29.42.16	56881	ESTABLISHED	3312	OUTLOOK.EXE	N/A


Progress:    0.00		Scanning VmwareLayer using PageMapScanner

Progress:    0.00		Scann

In [4]:
python3 ../../volatility3/vol.py -f ../S039/Memory/ACC-03-dc8c73f9.vmem windows.netscan.NetScan

Volatility 3 Framework 2.0.0

Offset	Proto	LocalAddr	LocalPort	ForeignAddr	ForeignPort	State	PID	Owner	Created

0x13d01e7d0	TCPv4	0.0.0.0	49187	0.0.0.0	0	LISTENING	2084	svchost.exe	-
0x13d0311e0	TCPv4	0.0.0.0	3389	0.0.0.0	0	LISTENING	340	svchost.exe	-
0x13d035cf0	TCPv4	-	56025	192.228.79.201	80	CLOSED	340	svchost.exe	-
0x13d069bb0	UDPv4	0.0.0.0	123	*	0		972	svchost.exe	2021-06-23 13:06:45.000000 
0x13d069bb0	UDPv6	::	123	*	0		972	svchost.exe	2021-06-23 13:06:45.000000 
0x13d20fba0	TCPv4	127.0.0.1	49191	127.0.0.1	49192	ESTABLISHED	1492	python.exe	N/A
0x13d23a010	TCPv4	-	0	104.112.49.6	0	CLOSED	2372	taskhost.exe	N/A
0x13d23a5e0	TCPv4	172.17.11.190	49293	52.203.247.180	443	ESTABLISHED	3852	winlogbeat.exe	N/A
0x13d246140	TCPv4	127.0.0.1	49196	127.0.0.1	49195	ESTABLISHED	1492	python.exe	N/A
0x13d25c490	TCPv4	172.17.11.190	49300	184.30.81.191	443	CLOSE_WAIT	1012	jucheck.exe	N/A
0x13d267340	TCPv4	127.0.0.1	49195	127.0.0.1	49196	ESTABLISHED	1492	python.exe	N/A
0x13d2678b0	TCPv4	-	56125	23.47.1

0x13f56ecf0	TCPv4	-	56007	151.101.0.81	443	CLOSED	4876	iexplore.exe	-
0x13f5f4790	UDPv4	127.0.0.1	51662	*	0		4876	iexplore.exe	2021-06-23 21:22:54.000000 
0x13f607710	TCPv4	-	56187	173.223.156.140	443	CLOSED	4876	iexplore.exe	-
0x13f61f950	UDPv6	::1	1900	*	0		3504	svchost.exe	2021-06-23 14:05:20.000000 
0x13f66fcf0	TCPv4	-	55990	23.0.148.145	443	CLOSED	4876	iexplore.exe	-
0x13f7f6010	TCPv4	-	56182	104.97.199.151	443	CLOSED	4876	iexplore.exe	-
0x13f8012b0	TCPv4	-	56084	185.25.50.167	80	CLOSED	4640	InvoiceApplica	-
0x13f848480	TCPv4	-	56196	151.101.1.26	443	CLOSED	4876	iexplore.exe	-
0x13f898dc0	UDPv4	127.0.0.1	51067	*	0		6080	iexplore.exe	2021-06-23 17:34:03.000000 
0x13f8a0010	TCPv4	172.17.11.190	56001	185.25.50.167	443	CLOSED	4876	iexplore.exe	-
0x13f90ad70	UDPv4	127.0.0.1	52400	*	0		5184	iexplore.exe	2021-06-23 20:27:04.000000 
0x13f962830	UDPv4	127.0.0.1	55911	*	0		5136	iexplore.exe	2021-06-23 22:00:27.000000 
0x13f99c810	TCPv4	172.17.11.190	56224	190.211.254.154	80	CLOSED	4640	Invo

The `iexplore.exe (PID 4876)` communicating with `185.25.50.167` is suspicious.

In [5]:
whois 185.25.50.167

% IANA WHOIS server
% for more information on IANA, visit http://www.iana.org
% This query returned 1 object

refer:        whois.ripe.net

inetnum:      185.0.0.0 - 185.255.255.255
organisation: RIPE NCC
status:       ALLOCATED

whois:        whois.ripe.net

changed:      2011-02
source:       IANA

# whois.ripe.net

inetnum:        185.25.48.0 - 185.25.51.255
netname:        LT-BACLOUD-20130429
country:        LT
org:            ORG-ISIT5-RIPE
admin-c:        BAC2006-RIPE
tech-c:         BAC2006-RIPE
status:         ALLOCATED PA
mnt-by:         RIPE-NCC-HM-MNT
mnt-by:         BACLOUD-MNT
mnt-routes:     AS61272-MNT
mnt-domains:    AS61272-MNT
created:        2013-04-29T10:36:27Z
last-modified:  2018-02-04T18:49:40Z
source:         RIPE # Filtered
geoloc:         55.93333 23.31667

organisation:   ORG-ISIT5-RIPE
org-name:       Informacines sistemos ir technologijos, UAB
country:        LT
org-type:       LIR
address:        Pramones st. 15
address:        78137
address:        Siauli